src/bmeg/edge.py

```diff
@@ -23,36 +23,19 @@ def from_dict(cls, data):
         return None
 
```

Comment:
Instead of defining a `label` method for every type just use a class method:

```
    def label(self):
        lbl = self.__class__.__name__
        return lbl[:1].lower() + lbl[1:]
```